src/masquerade/providers/open_sgid.py

```diff
@@ -29,14 +29,14 @@ def get_suggestion_from_record(oid, full_address, address_system, city):
     return {'isCollection': False, 'magicKey': oid, 'text': f'{full_address}, {zone}'}
 
 
-def get_suggestions(search_text):
+def get_suggestions(search_text, max_results):
     """ queries the database and returns suggestions based on the input text
     """
     query = f'''
         select {','.join(out_fields)} from {ADDRESS_POINTS_TABLE}
-        where upper({FULL_ADDRESS_FIELD}) like upper(\'{search_text}%\')
+        where {FULL_ADDRESS_FIELD} ilike \'{search_text}%\'
```

Comment:
**thought**: should this be a starts with search or a contains search? `%text%` vs `text%`

Reply:
I definitely feel like it should be when I query the other datasets such as roads, cities, etc. But I didn't feel like it gave great results for a typeahead with addresses. I'm open to other opinions if you feel strongly.

Reply:
good point... different tables will have different styles of search.